2018-05-Berlin/attendee-rsvp.md

```diff
@@ -0,0 +1,145 @@
+1. [Mark Hinkle](https://github.com/mrhinkle) (Node Foundation)
+1. [Matteo Collina](https://github.com/mcollina) (Organizer)
+1. [Maya Lekova](https://github.com/MayaLekova)
+1. [Michael Dawson](https://github.com/mhdawson)
+1. [Michaël Zasso](https://github.com/targos)
+1. [Myles Borins](https://github.com/MylesBorins) (Organizer)
+1. [Nicola Del Gobbo](https://github.com/NickNaso)
+1. [Peter Marshall](https://github.com/psmarshall)
+1. [Rachel White](https://github.com/rachelnicole)- MAYBE
+1. [Rich Trott](https://github.com/Trott)
+1. [Richard Littauer](https://github.com/RichardLitt)- MAYBE
+1. [Ruben Bridgewater](https://github.com/BridgeAR)
+1. [Shelley Vohr](https://github.com/codebytere)
+1. [Stephen Belanger](https://github.com/Qard)
+1. [Thomas Watson](https://github.com/watson)
+1. [Tierney Cyren](https://github.com/bnb)
+1. [Till Schneidereit](https://github.com/tschneidereit)
+1. [Tobias Nießen](https://github.com/tniessen)
+1. [Tracy Hinds](https://github.com/hackygolucky) (Organizer)
+1. [Ujjwal Sharma](https://github.com/ryzokuken)
+1. [Vladimir de Turckheim](https://github.com/vdeturckheim)
+1. [Waleed Ashraf](https://github.com/WaleedAshraf)
+1. [Yang Guo](https://github.com/hashseed)
+1. [Yihong Wang](https://github.com/yhwang)
+1. [Yosuke Furukawa](https://github.com/yosuke-furukawa)
+1. [Yunong Xiao](https://github.com/yunong)
+1. [Zeke Sikelianos](https://github.com/zeke)
+
+### Attendees Remote
+
+There will be breakout sessions and we will do our best to optimize for remote
+attendance. More information will be posted on the agenda. Please let us know if
+you have any questions!
+
+1. [Bradley Meck](https://github.com/bmeck)
+1. [Colin Ihrig](https://github.com/cjihrig)
+1. [Gus Caplan](https://github.com/devsnek)
+1. [Liran Tal](https://github.com/lirantal)
+1. [Matheus Marchini](https://github.com/mmarchini)
+1. [Trivikram Kamat](https://github.com/trivikr)
+
+# Node Collaborator’s Dinner - 2018-05-31, 7pm
+
+Dinner will hosted by Node Foundation the first day of the Summit at Brauhaus
+Lemke, a short Uber ride from the AWS offices.
+
+```
+BRAUHAUS LEMKE
+AM HACKESCHEN MARKT
+DIRCKSENSTR. 143
+10178 BERLIN-MITTE
+```
+[Google Maps :round_pushpin:](https://goo.gl/8uFWUX)
+
+[Google Maps Direction from Venue :arrow_double_up:](https://goo.gl/a2JxXk)
+
+Please list if you have dietary restrictions below and submit a PR if you will
+be joining.
+
+### Guests
+
+1. [Abraham Agiri Jr](https://github.com/codeekage) - no dietary restrictions
+1. [Adam Miller](https://github.com/amiller-gh) - no dietary restrictions
+1. [Ali Ijaz Sheikh](https://github.com/ofrobots) - allergies to shellfish, no pork
+1. [Anatoli Papirovski](https://github.com/apapirovski) - no dietary restrictions
+1. [Anna Henningsen](https://github.com/addaleax) - vegetarian
+1. [Benedikt Meurer](https://github.com/bmeurer) - no pork, no wheat, no sugar
+1. [Benjamin Gruenbaum](https://github.com/benjamingr)
```

Comment:
@benjamingr can you add any dietary restrictions or specify none? 💃

Reply:
Sure, there are no restrictions - do you want me to PR that in once it gets merged?

Reply:
@benjamingr nm i'm all up in this rn so i'll add. :)